src/status_im/utils/db.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.utils.db
+  (:require [clojure.string :as string]
+            [cljs.spec.alpha :as spec]
+            [status-im.js-dependencies :as dependencies]))
+
+(defn address? [s]
+  (.isAddress dependencies/Web3.prototype s))
+
+(defn hex-string? [s]
```

Comment:
@jeluard these functions are not new https://github.com/status-im/status-react/blob/883b702fcba7d94b93828ccfaa6d6053100fe52d/src/status_im/ui/screens/contacts/db.cljs#L16